doc/translation_process.md

```diff
@@ -62,7 +62,7 @@ token =
 username = USERNAME
 ```
 
-The Transifex Bitcoin project config file is included as part of the repo. It can be found at `.tx/config`, however you shouldn’t need change anything.
+The Transifex Bitcoin project config file is included as part of the repo. It can be found at `.tx/config`, however you shouldn’t need  to change anything.
```

Comment:
You have an extra space here between "need" and "to"

Reply:
alright, will be changing it now